src/App.jsx

```diff
@@ -14,33 +16,93 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const ALPHABIT = 'alphabetically';
+const LENGTH = 'by length';
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+function getSortedList(goods, reverse, { sortField }) {
+  const preparedGoods = [...goods];
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+  if (sortField) {
+    preparedGoods.sort((good1, good2) => {
+      switch (sortField) {
+        case ALPHABIT:
```

Comment:
Not a mistake in code, but in English)))

Reply:
This is "custom English" )))
To avoid it - use https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker or similar